### ISSUE TITLE
[2.0] Only entity labels are considered for filters

### ISSUE THREAD

**Issue by strawgate** [legrego/homeassistant-elasticsearch#350]
Should we also check device labels for inclusion/exclusion of labels?

**Comment by legrego** [comment]
Yeah, this makes a lot of sense to me. Some devices can have many entities, so at times it's easier from an administrative perspective to "just" configure the device, and let the entities respect the device configuration.

**Comment by strawgate** [comment]
Would we consider this a breaking change and I should land this in 2.0?

**Comment by legrego** [comment]
@strawgate I suppose we should consider this breaking, so landing this in 2.0 makes sense to me.
